airflow/contrib/hooks/spark_submit_hook.py

```diff
@@ -241,7 +242,9 @@ def _process_log(self, itr):
         """
         # Consume the iterator
         for line in itr:
-            line = line.decode('utf-8').strip()
+            if isinstance(line, six.binary_type):
```

Comment:
For inspiration see: https://stackoverflow.com/questions/38181494/what-is-the-difference-between-using-universal-newlines-true-with-bufsize-1-an